### DISCUSSION TITLE
Wi-Fi 6 Adapters

### DISCUSSION THREAD

**Discussion by Tockman** [chris1111/Wireless-USB-Big-Sur-Adapter#124] [General]
Hello! Will your remarkable utility support wi-fi 6 USB adapters, e.g. this one: https://www.dlink.com/en/products/dwa-x1850-ax1800-wi-fi-6-usb-adapter ? AFAIK it has no any macOS driver, and pity is there's no info on chipset type it's built on. Thank you.

**Comment by Tockman** [comment]
It may be supposedly RTL8832AU, RTL8852AU or RTL8852BU chipset.

**Reply by chris1111** [reply]
> It may be supposedly RTL8832AU, RTL8852AU or RTL8852BU chipset.


See [Download](https://support.dlink.ca/ProductInfo.aspx?m=DWA-X1850) extract the drivers from Windows and you will see what it is because No Infos on PDF about it

[DWA-X1850_REVA_DATASHEET_1.01.PDF](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/files/6868541/DWA-X1850_REVA_DATASHEET_1.01.PDF)
[DWA-X1850_REVA_QIG_1.00_EN.PDF](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/files/6868542/DWA-X1850_REVA_QIG_1.00_EN.PDF)

**Reply by Tockman** [reply]
I read both of them so I know it. But I'm aware the corresponding driver supports three chipsets I noticed, so it's logical to assume that DWA-X1850 is built on one of them (see https://www.addfiles.ru/news/realtek_rtl8832au_wifi_6_usb_adapter_driver_5001_0_13_103_whql_drajver_dlja_wi_fi_802_11ax_pod_windows/2021-07-20-3102). The problem is there's no macOS drivers for any of those chipsets yet...

**Comment by chris1111** [comment]
Yea the WIFI manufacturer exclude macOS from the adapter since Catalina thats why I create the program and there is no source from Realtek for the drivers.
Maybe a control freak from apple on WIFI manufacturer ; same like NVIDIA 😆  A lots of WIFI compagnie  link the user with a old adapter to this program here
